lib/ClangImporter/ImportType.cpp

```diff
@@ -538,11 +540,8 @@ namespace {
           hint = ImportHint::CFPointer;
         } else if (mappedType->isAnyExistentialType()) { // id, Class
           hint = ImportHint::ObjCPointer;
-        } else if (type->isBlockPointerType()) {
-          // FIXME: This should eventually be "isAnyPointerType", but right now
-          // non-object, non-block pointers are never Optional in Swift; they
-          // just can have a value of 'nil' themselves.
-          hint = ImportHint::CustomNullablePointer;
+        } else if (type->isPointerType() || type->isBlockPointerType()) {
```

Comment:
Note for reviewers: the comment was wrong. Clang's `isAnyPointerType` includes pointers-to-ObjC-objects (covered above) and does *not* include block pointers.